scripts/globals/conquest.lua

```diff
@@ -1386,4 +1386,4 @@ function conquestUpdate(zone, player, updateType, messageBase)
             end
         end
     end
-end;
+end;
```

Comment:
No newline, what world we live in.

Reply:
I'd say something about how we should leave those whatever they are as we find'em but my auto format tools keep adding it without my noticing so I'd be a hypocrite ^^ some day I'll look into isolating that feature from my dsp local copy >.>